libraries/botbuilder-ai-orchestrator/package.json

```diff
@@ -32,7 +32,7 @@
     "botbuilder-dialogs": "4.1.6",
     "botbuilder-dialogs-adaptive": "4.1.6",
     "botbuilder-dialogs-declarative": "4.1.6",
-    "orchestrator-core": "4.12.0-preview",
+    "orchestrator-core": "next",
```

Comment:
We intentionally removed this right? I thought we wanted to pin to a specific version.

Reply:
You can leave this - on Sunday, i'll post a new PR with our RC build.

Reply:
Discussed, we'll pin this in `4.13` branch.